src/v2/Components/Authentication/Views/SignUpForm.tsx

```diff
@@ -11,7 +11,7 @@ import { Formik, FormikProps } from "formik"
 import { Component } from "react"
 import * as React from "react"
 import { recaptcha } from "v2/Utils/recaptcha"
-import { data as sd } from "sharify"
+import { data as sd } from "sharify" // eslint-disable-line no-restricted-imports
```

Comment:
Same as above: It seems out of scope to refactor this component to stop using sharify.

Reply:
(This can also go in this ticket here https://github.com/artsy/force/pull/9143/files#r794756516)